external-crates/move/crates/move-compiler/tests/linter/correct_freezing_capability.move

```diff
@@ -0,0 +1,25 @@
+module 0x42::M {
```

Comment:
nit, maybe these tests should be something like `allowed_freezing_capability` and `disallowed_freezing_capability` 

I am not sure `correct` really covers what is going on here, especially since it is just a heuristic

Reply:
refactored